src/sass/partials/components/_content-card.scss

```diff
@@ -0,0 +1,30 @@
+.card {
```

Comment:
If the file name is `content-card` this component should be `.content-card`, not `.card`. Card is a little _too_ generic, maybe we want a more descriptive name, like `basic-card` or `minimal-card`